content/privacy-policy.md

```diff
@@ -0,0 +1,139 @@
+
+Your consent to this Privacy Policy followed by Your submission of such information represents Your agreement to that transfer.
+
+## Disclosure of Your Usage Data
+
+## Law enforcement
+
+Under certain circumstances, the Company may be required to disclose Your Usage Data if required to do so by law or in response to valid requests by public authorities (e.g. a court or a government agency).
+
+## Other legal requirements
+
+The Company may disclose Your Usage Data in the good faith belief that such action is necessary to:
+
+* Comply with a legal obligation
+* Protect and defend the rights or property of the Company
+* Prevent or investigate possible wrongdoing in connection with the Service
+* Protect the personal safety of Users of the Service or the public
+* Protect against legal liability
+
+## Security of Your Personal Data
+
+The security of Your Data is important to Us, but remember that no method of transmission over the Internet, or method of electronic storage is 100% secure. While We strive to use commercially acceptable means to protect Your Data, We cannot guarantee its absolute security.
+
+## Children's Privacy
+
+Our Service does not address anyone under the age of 13. We do not knowingly collect personally identifiable information from anyone under the age of 13. If You are a parent or guardian and You are aware that Your child has provided Us with Personal Data, please contact Us. If We become aware that We have collected Personal Data from anyone under the age of 13 without verification of parental consent, We take steps to remove that information from Our servers.
+
+If We need to rely on consent as a legal basis for processing Your information and Your country requires consent from a parent, We may require Your parent's consent before We collect and use that information.
+
+## Links to Other Websites
+
+Our Service may contain links to other websites that are not operated by Us. If You click on a third party link, You will be directed to that third party's site. We strongly advise You to review the Privacy Policy of every site You visit.
+
+We have no control over and assume no responsibility for the content, privacy policies or practices of any third party sites or services.
+
+## Changes to this Privacy Policy
+
+We may update Our Privacy Policy from time to time. We will notify You of any changes by posting the new Privacy Policy on this page. We will provide a prominent notice on Our Service, prior to the change becoming effective and update the "Last updated" date at the top of this Privacy Policy.
+
+You are advised to review this Privacy Policy periodically for any changes. Changes to this Privacy Policy are effective when they are posted on this page.
+
+## Contact Us
+
+If you have any questions about this Privacy Policy, You can contact us:
+
+MTU XRP Ledger Trust
+
+Regd Office
+
+15, Ringtee
+
+Kuressaare
+
+Estonia 93815
```

Comment:
You can use two trailing spaces to force a linebreak (not paragraph break) in Markdown. That plus bold makes this read a little better:

```suggestion
MTU XRP Ledger Trust

**Regd Office**  
15, Ringtee  
Kuressaare  
Estonia 93815
```